src/components/molecules/AdminSection/AdminSection.tsx

```diff
@@ -28,7 +28,7 @@ export const AdminSection: React.FC<AdminSectionProps> = ({
   );
 
   return (
-    <section className="AdminSection">
+    <section className="mb-6 mt-6 flow-root">
```

Comment:
should probably do a copy-paste these atom/molecule components into `./src/components/admin/` and give them the full TW treatement while removing any `.module.scss` file

Reply:
All `AdminSection` components have been reverted to their original state and they have been extracted into new ones located at `/admin/...`